packages/core/src/components/Checkbox/Checkbox.tsx

```diff
@@ -40,6 +40,7 @@ const Checkbox = ({
   }, [externalState]);
 
   const checkboxId = id || autoGenId;
+  const register = rest.register;
```

Comment:
Do we need pass `register` to the CheckboxBase? If not, could we get register in the Checkbox props?
I mean the next approach:
`const Checkbox = ({ text, id, ...other props..., register, ...rest }: ICheckboxProps)`

Reply:
Done